MAINTAINERS.md

```diff
@@ -0,0 +1,14 @@
+# The Piraeus Datastore Maintainers
+
+This file lists the maintainers of the entire Piraeus Datastore project, hosted and managed by the
+[Piraeus Datastore Organization](https://github.com/piraeusdatastore). The responsibilities of maintainers are listed in
+the [GOVERNANCE.md](./GOVERNANCE.md) file
+
+## Maintainers
```

Comment:
maybe add some "in alphabetical order" to make clear this does not imply anything else than that.